### DISCUSSION TITLE
YOLOv7 training in darknet framework

### DISCUSSION THREAD

**Discussion by BenJam29** [AlexeyAB/darknet#8709] [Q&A]
Hi everyone, 

I have a question about feasibility of training YOLOv7 models in darkent framework rather than Pytorch.
I just saw the following files for yolov7 in the release notes but I am not sure we can do a full training and model creating using Yolov7 inside darknet efficiently.

YOLOv7x:
cfg: https://raw.githubusercontent.com/AlexeyAB/darknet/master/cfg/yolov7x.cfg
weights: https://github.com/AlexeyAB/darknet/releases/download/yolov4/yolov7x.weights
weights for fine-tuning: https://github.com/AlexeyAB/darknet/releases/download/yolov4/yolov7x.conv.147

For example, can we do something like the following for yolov7?
`darknet.exe detector train data/obj.data yolov7.cfg yolov7.conv.147`

Thank you for your help,

**Comment by MinfengHung** [comment]
Hi, you may refer to this
[Official YOLOv7: Trainable bag-of-freebies sets new state-of-the-art for real-time object detectors #8595](https://github.com/AlexeyAB/darknet/issues/8595)
